drivers/include/net/netdev.h

```diff
@@ -281,6 +281,32 @@ struct netdev_radio_rx_info {
     uint8_t lqi;        /**< LQI of a received frame */
 };
 
+/**
+ * @name    Flags indicating which fields in @ref netdev_tx_info_t
```

Comment:
```suggestion
 * @brief    Flags indicating which fields in @ref netdev_tx_info_t
```

Reply:
For groups `@name` is correct. That group only has a single member as of now, but that may change in the future

Reply:
Doxygen does not like it though - or you need additional doc for that member